src/lib/Components/Inbox/Conversations/Message.tsx

```diff
@@ -94,6 +96,18 @@ export class Message extends React.Component<Props, any> {
     })
   }
 
+  renderBody() {
```

Comment:
pretty simple, if this is the first message, it renders the initial text, otherwise just the body